monitoring/count-items-cronjob/alerts.yaml

```diff
@@ -0,0 +1,29 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: count_items_cronjob
+    type: constant
+    value: artesca-data-ops-count-items
+  - name: count_items_job_duration_threshold
+    type: config
+    value: 3600
+
+groups:
+  - name: count-items-cronjob/alerts.rules
+    rules:
+      - alert: CountItemsJobTakingTooLong
+        Expr: |
+          time() -
+          (sum by(job_name) (kube_job_status_failed{job_name=~"${count_items_cronjob}.*"})
+          > sum by(job_name) (kube_job_status_completion_time{job_name=~"${count_items_cronjob}.*"})
+          or sum by(job_name) (kube_job_status_completion_time{job_name=~"${count_items_cronjob}.*"}))
+          > ${count_items_job_duration_threshold}
+        Labels:
+          severity: warning
+        Annotations:
+          description: |
+            Job ${count_items_cronjob} is taking more than 1h to complete.
+            This means that consumption information will be delayed
+            and the reported metrics might not be accurate anymore.
+          summary: count-items cronjob didn't finish after 1h
```

Comment:
As we can change the frequency of this job (this will be documented very soon for the veeam use case, with warnings of course), I would suggest, if possible, to have a dynamic  `1h` based on the current frequency of the cronjob. If changing the string is implying too much work, we could rewrite it not to mention the duration?

Reply:
yeah good idea! I will address that

Reply:
it should be possible to use the variables (count_items_job_duration_threshold) in the description ;
and I am not sure, but it _may_ be possible  to use functions or operations in the template, something like `{{ ${count_items_job_duration_threshold} / 3600 }}`

Reply:
seems not working the '/' operand
```
/tests/monitoring/update-bucket-capacity-info-cronjob/alerts.rendered.yaml: group "update-bucket-capacity-info-cronjob/alerts.rules", rule 1, "UpdateBucketCapacityJobTakingTooLong": annotation "description": template: __alert_UpdateBucketCapacityJobTakingTooLong:2: unexpected "/" in operand
```

Reply:
or I just leave it `taking more than 3600s to complete`